backend/routers/task_integrations.py

```
@@ -0,0 +1,1218 @@
+            # Create task in Todoist
+            body = {
+                'content': request.title,
+                'priority': 2,
+            }
+            if request.description:
+                body['description'] = request.description
+            if request.due_date:
+                due = datetime.fromisoformat(request.due_date.replace('Z', '+00:00'))
+                body['due_string'] = due.strftime('%Y-%m-%d')
+
+            response = requests.post(
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)